data_validation/config_manager.py

```diff
@@ -1008,6 +1019,66 @@ def _get_comparison_max_col_length(self) -> int:
             )
         return self._comparison_max_col_length
 
+    def _get_source_raw_data_types(self) -> dict:
+        """Return a dict of column name: raw data type.
+
+        The raw data type is the source/target engine type, for example it might
+        be "NCLOB" when the Ibis type states "string"."""
+        if self._source_raw_data_types is None:
+            if clients.is_oracle_client(self.source_client):
+                raw_data_types = self.source_client.raw_metadata(
+                    database=self.source_schema,
+                    table=self.source_table,
+                    query=self.source_query,
+                )
+                self._source_raw_data_types = {
+                    _.casefold(): raw_data_types[_] for _ in raw_data_types
+                }
+            else:
+                self._source_raw_data_types = {}
+        return self._source_raw_data_types
+
+    def _get_target_raw_data_types(self) -> dict:
+        """Return a dict of column name: raw data type.
+
+        The raw data type is the source/target engine type, for example it might
+        be "NCLOB" when the Ibis type states "string"."""
+        if self._target_raw_data_types is None:
+            if clients.is_oracle_client(self.target_client):
+                raw_data_types = self.target_client.raw_metadata(
+                    database=self.target_schema,
+                    table=self.target_table,
+                    query=self.target_query,
+                )
+                self._target_raw_data_types = {
+                    _.casefold(): raw_data_types[_] for _ in raw_data_types
+                }
+            else:
+                self._target_raw_data_types = {}
+        return self._target_raw_data_types
+
+    def _is_oracle_lob(self, casefold_column_name: str) -> bool:
+        return bool(
```

Comment:
```suggestion
         """Checks if a column in either the source or target database is an Oracle LOB (Large Object) data type. 
 
        Args:
          casefold_column_name: The case-insensitive name of the column to check."""
        return bool(
```

I don't know why the suggestion block is formatting the suggested documentation string with code syntax, but you can get the idea 😅